Automattic/wp-calypso
# Hosting Metrics: Modernize the header

Fixes https://github.com/Automattic/dotcom-forge/issues/3389

## Proposed Changes

Modernizes the header according to design.

### Desktop

<img width="1575" alt="CleanShot 2023-08-10 at 15 34 12@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/116170b7-6af0-4106-a5ca-a824b523fd2e">

### Mobile

<img width="583" alt="CleanShot 2023-08-10 at 15 36 37@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/f08b3838-ccad-4ec9-95fd-1feb205a99e9">

## Testing Instructions

1. Navigate to `/site-monitoring/<site>`.
2. Verify the header appears expected.
